components/assetcard/index.css

```diff
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
 /* outer container, unstyled */
 .spectrum-AssetCard {
   /* todo: this isn't quite the size from the XD file as 232px is not a size token, so we use 224px */
-  --spectrum-assetcard-asset-size: var(--spectrum-global-dimension-static-size-2800);
+  --spectrum-assetcard-asset-size: 224px;
+  --spectrum-assetcard-background-color: var(--spectrum-gray-200);
+  --spectrum-assetcard-asset-animation-duration: var(--spectrum-animation-duration-100);
+  --spectrum-assetcard-assetcontainer-border-size: 1px;
+  --spectrum-assetcard-header-margin-block-start: var(--spectrum-spacing-300);
 
-  --spectrum-assetcard-overlay-background-color: var(--spectrum-alias-assetcard-overlay-background-color);
-  --spectrum-assetcard-border-color-selected: var(--spectrum-alias-assetcard-border-color-selected);
-  --spectrum-assetcard-border-color-selected-hover: var(--spectrum-alias-assetcard-border-color-selected-hover);
-  --spectrum-assetcard-border-color-selected-down: var(--spectrum-alias-assetcard-border-color-selected-down);
-  --spectrum-assetcard-selectionindicator-background-color-ordered: var(--spectrum-alias-assetcard-selectionindicator-background-color-ordered);
-
-  --spectrum-assetcard-border-radius: var(--spectrum-alias-border-radius-regular);
+  --spectrum-assetcard-border-radius: var(--spectrum-corner-radius-100);
   --spectrum-assetcard-border-color: transparent;
-  --spectrum-assetcard-border-color-hover: var(--spectrum-alias-border-color-hover);
-  /* todo: this was hardcoded, it should reference --spectrum-alias-border-color-down once that becomes +200 */
-  --spectrum-assetcard-border-color-down: var(--spectrum-global-color-gray-600);
-
-  --spectrum-assetcard-background-color: var(--spectrum-global-color-gray-200);
-
-  --spectrum-assetcard-focus-ring-gap: var(--spectrum-global-dimension-static-size-65);
-  --spectrum-assetcard-focus-ring-border-radius: var(--spectrum-alias-focus-ring-border-radius-regular);
-
-  --spectrum-assetcard-selectionindicator-background-color-default: var(--spectrum-alias-background-color-quickactions);
-  --spectrum-assetcard-selectionindicator-margin: var(--spectrum-global-dimension-size-150);
-  --spectrum-assetcard-selectionindicator-size: var(--spectrum-global-dimension-static-size-500);
-  --spectrum-assetcard-selectionindicator-border-radius: var(--spectrum-alias-border-radius-regular);
-
-  --spectrum-assetcard-heading-text-color: var(--spectrum-global-color-gray-900);
-  --spectrum-assetcard-content-text-color: var(--spectrum-global-color-gray-700);
+  --spectrum-assetcard-border-color-hover: var(--spectrum-gray-500);
+  --spectrum-assetcard-border-color-down: var(--spectrum-gray-600);
+
+  --spectrum-assetcard-focus-ring-gap: 5px;
```

Comment:
Would this map well to: `var(--spectrum-focus-indicator-gap)`?

Reply:
I wanted it to! But the values are different `--spectrum-focus-indicator-gap: 2px;`